core/src/main/java/com/linecorp/armeria/common/util/Version.java

```diff
@@ -111,7 +111,7 @@ public static Version get(String artifactId, ClassLoader classLoader) {
 
     /**
      * Retrieves the version information of Armeria artifacts.
-     * This method is a shortcut for {@code identify(Version.class.getClassLoader())}.
+     * This method is a shortcut for {@linkplain #getAll(ClassLoader) getAll(Version.class.getClassLoader())}.
```

Comment:
`@link` is better than `@linkplain` in this case because the text is code.

Reply:
Updated.